src/ui/Cta/Cta.css

```
@@ -0,0 +1,5 @@
+.Cta {
```

P3:
이 클래스명도 다른 클래스명처럼 소문자로 시작하면 좋겠네요.
```suggestion
.cta {
```